src/components/SearchResults.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+const iconNames = {
+  Yes: '/static/check-circle.svg',
+  'In Moderation': '/static/alert-circle-yellow.svg',
+  Harmful: '/static/alert-circle-red.svg',
+  Toxic: '/static/x-circle.svg'
+};
+
+const Results = ({ results }) => {
+  if (results === '') return null;
```

Comment:
[very minor/code style] It's usually preferred to add an empty line after a guard clause such as this. Up to you, though.